ale_linters/elixir/mix.vim

```diff
@@ -32,11 +35,13 @@ endfunction
 function! ale_linters#elixir#mix#GetCommand(buffer) abort
     let l:project_root = ale#handlers#elixir#FindMixProjectRoot(a:buffer)
 
-    let l:temp_dir = ale#command#CreateDirectory(a:buffer)
+    let l:build_dir = ale#Var(a:buffer, 'elixir_mix_random_build_dir')
+    \   ? ale#command#CreateDirectory(a:buffer)
+    \   : ale#Var(a:buffer, 'elixir_mix_build_dir')
```

Comment:
Could you add code to create the other directory if it doesn't exist already?

Add tests for this.

Reply:
I think mix will create the directory for you. I haven't had to explicitly create it in my testing.

I didn't see any tests for this plugin. I'm not quite sure where to start. Do you think tests are a blocker for merging this?

Reply:
Yeah, you should add some tests for this. Have a look at some existing tests for examples, and `:help ale-dev` for documentation.